app/components/item-details/fonts.css

```diff
@@ -0,0 +1,11 @@
+@font-face
+{
+    font-family: "GothamProMedium";
```

Comment:
Let's not include font weight properties (like `medium` or `bold`) in family names: it does not scale well. Better specify `font-size` instead.